File: output/ChatGPT/codenet/Python/Java/s622795537.java
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;
import java.util.ArrayList;
import java.util.Collections;

public class s622795537 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        scanner.nextLine();
        Map<String, Integer> arr = new HashMap<>();

        for (int i = 0; i < n; i++) {
            String s = scanner.nextLine();
            if (!arr.containsKey(s)) {
                arr.put(s, 1);
            } else {
                arr.put(s, arr.get(s) + 1);
            }
        }

        int maxCount = Collections.max(arr.values());
        ArrayList<String> dic = new ArrayList<>();
        for (Map.Entry<String, Integer> entry : arr.entrySet()) {
            if (entry.getValue() == maxCount) {
                dic.add(entry.getKey());
            }
        }

        Collections.sort(dic);
        for (String s : dic) {
            System.out.println(s);
        }
    }
}

File: output/ChatGPT/codenet/Python/C++/s622795537.cpp
#include <iostream>
#include <map>
#include <vector>
#include <algorithm>

int main() {
    int n;
    std::cin >> n;

    std::map<std::string, int> arr;

    for (int i = 0; i < n; i++) {
        std::string s;
        std::cin >> s;

        if (arr.find(s) == arr.end()) {
            arr[s] = 1;
        } else {
            arr[s] += 1;
        }
    }

    int max_count = 0;
    for (auto it = arr.begin(); it != arr.end(); ++it) {
        max_count = std::max(max_count, it->second);
    }

    std::vector<std::string> dic;
    for (auto it = arr.begin(); it != arr.end(); ++it) {
        if (it->second == max_count) {
            dic.push_back(it->first);
        }
    }

    std::sort(dic.begin(), dic.end());

    for (auto it = dic.begin(); it != dic.end(); ++it) {
        std::cout << *it << std::endl;
    }

    return 0;
}